engines/director/lingo/lingo-object.cpp

```diff
@@ -907,8 +910,21 @@ bool BitmapCastMember::setField(int field, const Datum &d) {
 		warning("STUB: BitmapCastMember::setField(): Unprocessed setting field \"%s\" of cast %d", g_lingo->field2str(field), _castId);
 		return false;
 	case kTheRegPoint:
-		warning("STUB: BitmapCastMember::setField(): Unprocessed setting field \"%s\" of cast %d", g_lingo->field2str(field), _castId);
-		return false;
+	{
+		if (d.type != POINT) {
+			warning("BitmapCastMember::setField(): Wrong Datum type %d for kTheRegPoint", d.type);
+			return false;
+		}
+		auto temp = _img;
+		_modified = true;
+		g_director->getCurrentMovie()->getScore()->renderSprites(g_director->getCurrentMovie()->getScore()->getCurrentFrame(), kRenderForceUpdate);
```

Comment:
To shorten this line I'd store the result of `g_director->getCurrentMovie()->getScore()` in a pointer first.